.github/workflows/build.yml

```diff
@@ -45,15 +45,14 @@ jobs:
         env:
           cache-name: cache-node-modules
         with:
-          path: ~/.npm
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package.json') }}
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
Since we're installing from the lock file with `npm ci`, every guide also recommended hashing on the lock file and not the base file. This also has the advantage that tweaks to development scripts and metadata won't impact the build cache.

Reply:
In short windows was caching nothing npm related, neither local dependencies nor global ones.